cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/dslReferenceContent.groovy

```diff
@@ -0,0 +1,151 @@
+package org.jenkinsci.plugins.workflow.cps.Snippetizer
+
+import org.jenkinsci.plugins.workflow.cps.GlobalVariable
+import org.jenkinsci.plugins.workflow.cps.Snippetizer
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor
+import org.jenkinsci.plugins.workflow.structs.DescribableHelper
+
+import javax.servlet.RequestDispatcher
+
+Snippetizer snippetizer = my;
+
+def l = namespace(lib.LayoutTagLib)
+def st = namespace("jelly:stapler")
+
+
+h1(_("Steps"))
+for (StepDescriptor d : snippetizer.getStepDescriptors(false)) {
```

Comment:
I share a similar sentiment with @jglick that this view contains too much logic that ideally should be in Java classes, then Groovy views can be used to render them, instead of `String` in Java, which is what @abayer is saying.

But such refactoring can come in later.